pkg/machines/app.jsx

```diff
@@ -128,6 +134,42 @@ class App extends React.Component {
 
         const pathVms = path.length == 0 || (path.length > 0 && path[0] == 'vms');
 
+        let vmContent;
+        if (path.length > 0 && path[0] == 'vm') {
+            const vm = vms.find(vm => vm.name == cockpit.location.options.name && vm.connectionName == cockpit.location.options.connection);
+            if (!vm)
+                return null;
+
+            const connectionName = vm.connectionName;
+            const expandedContent = vm.isUi ? undefined : (
```

Comment:
What does `isUi` actually do? I see it being set in `setVmCreateInProgress()` and `setVmInstallInProgress()`, so it sounds like some kind of "pending operation" flag?

**Update**: Ah, I get it now -- that's kind of a dummy placeholder until libvirt gets a real domain object for newly created VMs.

Reply:
> dummy placeholder until libvirt gets a real domain object for newly created VMs

This should probably be turned into a comment above `.isUI` usage in the code.